os/drivers/audio/alc5658.c

```diff
@@ -1050,13 +1099,13 @@ static int alc5658_enqueuebuffer(FAR struct audio_lowerhalf_s *dev, FAR struct a
 	if (!priv->running) {
 		/* Add the new buffer to the tail of pending audio buffers */
 		alc5658_takesem(&priv->devsem);
-		sq_addlast((sq_entry_t*)&apb->dq_entry, &priv->pendq);
+		sq_addlast((sq_entry_t *)&apb->dq_entry, &priv->pendq);
```

Comment:
what is correct? (a *)b or (a*)b ?

Reply:
This is modified by our formatter script.

Reply:
Please Adjust format Manually